src/js/landing.js

```diff
@@ -159,6 +159,7 @@ onRender(
       iframeNode.setAttribute('frameborder', '0');
       iframeNode.setAttribute('scrolling', 'no');
       iframeNode.setAttribute('title', 'nightwatch-substack');
+      iframeNode.setAttribute('class', 'footer__wrapper-inner-social-subscribe-form');
```

Comment:
Just changing the height to `160px` here instead of `125px` seems to fix the issue. We'd rather do that than to change the content and logo in the footer (which if you notice is similar to what we have in the header).

Reply:
Yeah. That's true. 😅

But it could be better on mobile. Can I change some styling? So that it looks good on mobile.
<img width="357" alt="Screenshot 2024-03-11 at 3 55 23 PM" src="https://github.com/nightwatchjs/nightwatch-www/assets/133094641/70cd14d4-773c-458a-beb6-c117bbe80323">

Reply:
Sure, you can try. But probably in a separate PR? I think the issue related to substack widget can be solved fairly easily.

Reply:
Okay. 
Can I keep PR for this simple fix?

Reply:
Sorry, I unthinkingly deleted the forked repository.

Reply:
Here is the new [PR](https://github.com/nightwatchjs/nightwatch-www/pull/87)